src/sources/socket/mod.rs

```diff
@@ -117,22 +125,21 @@ mod test {
     use super::SocketConfig;
     use crate::event;
     use crate::runtime;
+    use crate::shutdown::{ShutdownCoordinator, ShutdownSignal};
     use crate::test_util::{
         block_on, collect_n, next_addr, send_lines, send_lines_tls, wait_for_tcp,
     };
     use crate::tls::{TlsConfig, TlsOptions};
     use crate::topology::config::{GlobalOptions, SourceConfig};
-    use futures01::sync::mpsc;
+    use futures01::sync::{mpsc, oneshot};
     use futures01::Stream;
     #[cfg(unix)]
     use futures01::{Future, Sink};
+    use std::net::UdpSocket;
     #[cfg(unix)]
     use std::path::PathBuf;
-    use std::{
-        net::{SocketAddr, UdpSocket},
-        thread,
-        time::Duration,
-    };
+    use std::time::Instant;
```

Comment:
minor nit: Can we merge this import with the one below?

Reply:
Done